src/index.html

```diff
@@ -5,9 +5,84 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <div data-qa="card" class="card">
+
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+        >
+
+      <p class="card__text"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__text-description">Product code: 195434</p>
+
+      <div class="card__review">
+        <div class="card__review card__review--stars">
+          <div class="
+          card__review
+          card__review--stars
+          card__review--stars-star"
+          >
+          </div>
+          <div class="
+          card__review
+          card__review--stars
+          card__review--stars-star"
+          >
+          </div>
+          <div class="
+          card__review
+          card__review--stars
+          card__review--stars-star"
+          >
+          </div>
+          <div class="
+          card__review
+          card__review--stars
+          card__review--stars-star"
+          >
+          </div>
+          <div class="
+          card__review
+          card__review--stars
+          card__review--stars-star"
+          >
+          </div>
```

Comment:
here you need to use block from stars task

Reply:
I changed their classes according to the code I had so I could use them in sass. If you don't need all the styles to be nested in each other, then I will rewrite the code so that it looks simpler and for the stars I will use the same code as in the previous task.